docs/lib/datastore/fragments/ios/getting-started/30_platformIntegration.md

```diff
@@ -1,31 +1,20 @@
-The `Amplify/Tools` pod provides an integration layer between the Amplify CLI headless mode (i.e. no AWS credentials needed) and Xcode. It integrates into the build process of the IDE to provide a seamless experience.
+The `amplify-app` library provides an integration layer between the Amplify CLI headless mode (i.e. no AWS credentials needed) and Xcode.
 
-First add the pod to your `Podfile`.
+First make sure you have the latest version installed:
 
-```ruby
-pod 'Amplify/Tools'
+```bash
+npm install -g amplify-app
```

Comment:
Do we need to split this out into two commands? why not just `npx amplify-app` ?

Reply:
to add to what Diego said, `npx amplify-app` is also slower and it doesn't replace older version that are possibly already installed.